packages/angular_devkit/build_angular/src/webpack/plugins/suppress-entry-chunks-webpack-plugin.ts

```diff
@@ -14,6 +14,11 @@ export class SuppressExtractedTextChunksWebpackPlugin {
   apply(compiler: import('webpack').Compiler): void {
     compiler.hooks.compilation.tap('SuppressExtractedTextChunks', (compilation) => {
       compilation.hooks.chunkAsset.tap('SuppressExtractedTextChunks', (chunk, filename) => {
+        if (compiler.options?.devServer?.hot) {
```

Comment:
This check will run for every chunk.  Can this be moved into the Webpack configuration generation so that the plugin could be added conditionally?